src/main/java/uk/gov/companieshouse/api/accounts/util/ixbrl/ixbrlgenerator/DocumentGeneratorConnection.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.companieshouse.api.accounts.util.ixbrl.ixbrlgenerator;
+
+public class DocumentGeneratorConnection {
+    private String requestMethod;
+    private String serviceURL;
+    private String requestBody;
+    private String authorizationProperty;
+    private String assetId;
+    private String contentType;
+    private String acceptType;
+    private String location;
+    private String templateName;
```

Comment:
I wouldn't expect templateName in a DocumentGeneratorConnection object. Can this be split into connect and response objects?